### PR TITLE
Completed email generator.

### PR DESCRIPTION
**Pull request by ocampd1**

### CODE REVIEW

File: chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts [hunk 1]
```diff
@@ -29,7 +31,7 @@ export class MockCustomerService {
     return firstName[Math.floor(Math.random() * firstName.length)];
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts]: Shouldn't you be returning the customer object here.

File: chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts [hunk 2]
```diff
@@ -8,10 +8,12 @@ export class MockCustomerService {
   constructor() { }
   public generateCustomer() {
     // TODO: create a new customer.
+    const firstName = this.generateFirstName();
+    const lastName = this.generateLastName();
     const newCust = {
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts]: Rename `newCust` variable to `newCustomer`.

File: chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts [hunk 3]
```diff
@@ -8,10 +8,12 @@ export class MockCustomerService {
   constructor() { }
   public generateCustomer() {
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts]: Please add a return type in method declaration.

File: chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts (hunk 2 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts]: Changed name from newCust to newCustomer.

File: chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts (hunk 3 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts]: Added return type method in declaration.

File: chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts (hunk 1 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts]: The first name is returning from the generateFirstName method.

File: chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts [hunk 4]
```diff
@@ -7,17 +7,20 @@ export class MockCustomerService {
...
+      firstName,
+      lastName,
+      email: this.generateEmail(firstName, lastName),
       phone: this.generatePhone(),
       city: this.generateCity(),
       state: this.generateState(),
       id: this.generateId()
     };
-    return newCust;
+    generateCustomer = newCustomer;
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts]: No need for this.  Just return `newCustomer`.

File: chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts [hunk 5]
```diff
@@ -7,17 +7,20 @@ export class MockCustomerService {
 
   constructor() { }
   public generateCustomer() {
-    // TODO: create a new customer.
-    const newCust = {
-      firstName: this.generateFirstName(),
-      lastName: this.generateLastName(),
-      email: this.generateEmail(),
+    let generateCustomer;
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts]: No need for this variable.  Please remove.

File: chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts [hunk 6]
```diff
@@ -7,17 +7,20 @@ export class MockCustomerService {
...
+      lastName,
+      email: this.generateEmail(firstName, lastName),
       phone: this.generatePhone(),
       city: this.generateCity(),
       state: this.generateState(),
       id: this.generateId()
     };
-    return newCust;
+    generateCustomer = newCustomer;
+    return generateCustomer;
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts]: Just return `newCustomer`.

File: chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts (hunk 4 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts]: Removed genCustomer.

File: chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts (hunk 5 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts]: Removed genCustomer

File: chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts (hunk 6 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts]: Reverted back for newCustomer.

Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts]: Completed all change requests

**Pull request merged**